### DISCUSSION TITLE
Pnpx no interactive

### DISCUSSION THREAD

**Discussion by meeroslav** [pnpm/pnpm#3539] [Q&A] [answered]
Since `pnpm` version 6, running `pnpx` requires confirmation:
```
> pnpx create-my-project@0.0.1
  ? Install the following package: create-my-project@0.0.1? (Y/n) › true
```

This stops me from using version 6 on CI. Is there a flag to disable interactive?

In v5 there was no user prompt while pnpx.

**Comment by zkochan** [comment, marked as answer]
`pnpx --yes`

**Reply by zkochan** [reply]
But you probably don't need pnpx:

https://pnpm.io/cli/exec

**Reply by meeroslav** [reply]
Thank you for the quick answer but this doesn't seem to be working. I still get prompt.
Not sure if this makes any difference but `create-my-project` is a custom package published to verdaccio registry.

**Reply by meeroslav** [reply]
Sorry, it works, just has to be set before the name, not after:
```
pnpx --yes create-my-project@0.0.1
```

Thank you!
